src/pages/index.js

```diff
@@ -0,0 +1,22 @@
+import Image from 'next/image'
+import { Inter } from 'next/font/google'
+import { Card } from '@/components';
+import { useCards } from '@/hooks';
+import { useCallback } from 'react';
+
+export default function Home() {
+  const { cards, handleCardClick, isCardsEnded, reset } = useCards();
+
+  const handleEmptyCardClick = useCallback((index) => () =>{
```

Comment:
Why do we return a function inside `useCallback` function?

Reply:
useCallback is really not working here, I'll fix it